ext/standard/string.c

```diff
@@ -5972,10 +5973,7 @@ PHP_FUNCTION(money_format)
 	zend_string *str;
 	ssize_t res_len;
 
-	ZEND_PARSE_PARAMETERS_START(2, 2)
-		Z_PARAM_STRING(format, format_len)
-		Z_PARAM_DOUBLE(value)
-	ZEND_PARSE_PARAMETERS_END();
+	PARSE_PARAMETERS_MONEY_FORMAT(format, format_len, value);
```

Comment:
How are we interpreting `format` and `format_len` as one argument? If the variables need to match the name, maybe we should document that.

Reply:
The generated definition of the `PARSE_PARAMETERS_MONEY_FORMAT` macro expects 3 parameters because I decided to use`Z_PARAM_STRING` for now for parsing string parameters:
```C
#define PARSE_PARAMETERS_MONEY_FORMAT(format, format_length, value) \
	ZEND_PARSE_PARAMETERS_START(2, 2) \
		Z_PARAM_STRING(format, format_length) \
		Z_PARAM_DOUBLE(value) \
	ZEND_PARSE_PARAMETERS_END();
```